### ISSUE TITLE
Coming in contact with creator

### ISSUE THREAD

**Issue by TheBonneSwe** [beawaksy/MessageBot#1]
Hi, sorry for opening an issue on this but I just couldn't find a way to PM you directly.

I've been looking into a messagebot for a gamingserver I host and I'm hoping for this one to help me with this.

This game, Euro Truck Simulator 2, is hosted via the game. That means that I need to have my game open to be able to host it, and while me or my dedicated PC host I could have it connected to chat at all time. 

I was wondering if you would mind help me understand how to set your bot up to spam diffrent qoutes during diffrent intervalls in my chat?

Basically all I want is it to choose diffrent quotes I've typed in, paste it into chat and press enter and do this every 5 - 10 minutes. It would mean a lot if you would help me with this but I understand if you don't got time and I'll search elsewhere. 

Regards.
